### PR TITLE
chore(i18n,curriculum): processed translations

### PR DESCRIPTION
**Pull request by camperbot**
This PR was opened auto-magically by Crowdin.

### CODE REVIEW

**Review by naomi-lgbt — DISMISSED** [review]

**Review by raisedadead — APPROVED** [review]

**Review by ojeytonwilliams — APPROVED** [review]

File: curriculum/challenges/espanol/06-quality-assurance/advanced-node-and-express/implementation-of-social-authentication-ii.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ passport-github should be required.
   );
 ```
 
-GitHub strategy should be setup correctly thus far.
+La estrategia de GitHub debe configurarse correctamente hasta ahora.
```
Comment (RafaelDavisH) [inline curriculum/challenges/espanol/06-quality-assurance/advanced-node-and-express/implementation-of-social-authentication-ii.md:73]:
```suggestion
La estrategia de GitHub debe estar configurada correctamente hasta ahora.
```

File: curriculum/challenges/espanol/06-quality-assurance/advanced-node-and-express/implementation-of-social-authentication-iii.md [hunk 2]
```diff
@@ -1,14 +1,14 @@
 ---
 id: 589a8eb3f9fc0f352b528e72
-title: Implementation of Social Authentication III
+title: Implementación de la autentificación social III
```
Comment (RafaelDavisH) [inline curriculum/challenges/espanol/06-quality-assurance/advanced-node-and-express/implementation-of-social-authentication-iii.md:3]:
```suggestion
title: Implementación de la autenticación social III
```

**Review by RafaelDavisH — COMMENTED** [review]
~A few suggestions, the rest~ LGTM

**Comment by RafaelDavisH** [comment]
I didn't realize these were Crowdin's translations.

**Pull request merged**